scripts/single-node.sh

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env bash
+
+# This script can be used in conjunction with
+# https://github.com/celestiaorg/celestia-app/blob/main/scripts/single-node.sh
```

Comment:
[for reviewers] if you want to try this locally, use https://github.com/celestiaorg/celestia-app/pull/3777